rules/code-quality/tests/Rector/Return_/SimplifyUselessVariableRector/Fixture/skip_special_class_name.php.inc

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace Rector\Naming\Tests\Rector\ClassMethod\RenameVariableToMatchNewTypeRector\Fixture;
+
+use Exception;
+
+final class SkipSpecialClassName
```

Comment:
can be `DoNotChangeSpecialClassName` or `DoNotImportSpecialClassName`

Reply:
Thanks, fixed it before I saw this.